src/BenchmarkDotNet/Portability/Memory/wmicMemoryInfoProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using BenchmarkDotNet.Helpers;
+using JetBrains.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BenchmarkDotNet.Portability.Memory
+{
+    /// <summary>
+    /// Memory information from output of the `wmic OS get TotalVisibleMemorySize, FreePhysicalMemory /Format:List` command.
+    /// Windows only.
+    /// </summary>
+    internal static class WmicMemoryInfoProvider
+    {
+        internal static readonly Lazy<MemoryInfo> WmicMemoryInfo = new Lazy<MemoryInfo>(Load);
+
+        [CanBeNull]
+        private static MemoryInfo Load()
+        {
+            if (RuntimeInformation.IsWindows())
+            {
+                string content = ProcessHelper.RunAndReadOutput("wmic", "OS get TotalVisibleMemorySize, FreePhysicalMemory /Format:List");
```

Comment:
Please, reuse the `wmic` call from `WmicCpuInfoProvider`

Reply:
`wmic cpu` is used for CPU and `wmic OS` is used for memory, the call output can not be reused

Reply:
Ok.